src/js/utils/useGlobalNav.js

```diff
@@ -3,7 +3,7 @@ import { load } from 'js-yaml';
 import { getNavFromConfig } from '../nav/globalNav';
 import sourceOfTruth from '../nav/sourceOfTruth';
 
-const appIds = ['application-services', 'openshift', 'insights', 'edge', 'ansible', 'settings'];
+const appIds = ['application-services', 'openshift', 'insights', 'ansible', 'subscriptions', 'cost-management', 'edge', 'settings'];
```

Comment:
Can you put the `edge` before `ansible`?
```suggestion
const appIds = ['application-services', 'openshift', 'insights', 'edge', 'ansible', 'subscriptions', 'cost-management', 'settings'];
```